nixos-modules/valheim.nix

```diff
@@ -61,6 +61,43 @@ in {
       description = lib.mdDoc "Whether to open ports in the firewall.";
     };
 
+    public = lib.mkOption {
+      type = lib.types.bool;
+      default = true;
+      description = lib.mdDoc ''
+        Toggles visibility on the Steam server & community lists.
+
+        When not set, defaults to true (visible).
+        Set to false to make the server private.
+      '';
+    };
+
+    logFile = lib.mkOption {
+      type = with lib.types; nullOr str;
+      default = null;
+      example = "/var/lib/valheim/log/valheim-server.log";
+      description = lib.mdDoc ''
+        The path where the server log file will be saved.
+
+        When set, this will redirect all logs from stdout to the specified path.
+        This means that the logs will not be captured by systemd's journal.
+        Leave this option unset to keep console logging enabled.
+
+        Make sure the valheim user has write access to the specified directory, otherwise
+        the valheim service fail to start and exit.
+      '';
+    };
```

Comment:
I think `-logFile` is there because upstream expects you to run it using their wrapper script, from the commandline, where the log output will just go to your terminal scrollback buffer.  The NixOS module provided by this flake instead runs it using systemd.  Telling the underlying program to log to a file *instead of* standard out/error somewhat goes against the purpose of running as a systemd service, so I would rather not expose that function.